src/router.js

```diff
@@ -186,6 +186,13 @@ const router = new Router({
     },
     { path: "*", redirect: "/app/" },
   ],
+  scrollBehavior(to, from, savedPosition) {
+    if (to.path === from.path) {
```

Comment:
Does this happen?

Reply:
Yes, if you use the pagination or search

Reply:
Ah, right, `path` doesn't include the query.